teachertool/src/components/styling/MakecodeFramePlaceholder.module.scss

```diff
@@ -0,0 +1,22 @@
+.makecode-frame-placeholder {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    color: var(--pxt-page-foreground);
+    opacity: 0.65;
+    font-size: 2rem;
```

Comment:
The text size is larger than the "Welcome to MakeCode Code Evaluation" text. I'm fine with leaving this as is if we want this text to be the first thing users look at, but if not, then I think we should make the text a bit smaller.

Reply:
Good point. Reduced to 1.5rem:

![image](https://github.com/user-attachments/assets/2dc5d982-133a-49ec-96ac-e95fc6b938f3)